README.md

```diff
@@ -74,3 +74,16 @@ Updates the given setting with the provided value for the meeting settings of th
 ## Contributing
 
 If you would like to make contributions to this plugin, please checkout the open issues labeled [`Help Wanted` and `Up For Grabs`](https://github.com/mattermost/mattermost-plugin-agenda/issues?q=is%3Aopen+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
+
+## Development
+
+This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/integrate/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/integrate/plugins/developer-setup/) for more information about developing and extending plugins.
+
+### Releasing new versions
+
+The version of a plugin is determined at compile time, automatically populating a `version` field in the [plugin manifest](plugin.json):
+* If the current commit matches a tag, the version will match after stripping any leading `v`, e.g. `1.3.1`.
+* Otherwise, the version will combine the nearest tag with `git rev-parse --short HEAD`, e.g. `1.3.1+d06e53e1`.
+* If there is no version tag, an empty version will be combined with the short hash, e.g. `0.0.0+76081421`.
+
+To disable this behaviour, manually populate and maintain the `version` field.
```

Comment:
The README should make clear what _this_ plugin is using. As I pointed out in another PR, I still think that version bump PRs are helpful, and we should keep them for Community-Supported plugins.

Reply:
Sure. The last sentence

> To disable this behaviour, manually populate and maintain the `version` field.

suggests that there are two ways to release a plugin: either with the `version` field or without it. The README should make it clear which way is used in this plugin and should not leave room for interpretation.

Reply:
@hanzei I have updated the README please have a look

Reply:
@hanzei Updated the readme for better clarity, please have a look